numpy/core/__init__.py

```diff
@@ -137,9 +137,32 @@ def _ufunc_reduce(func):
 import copyreg
 
 copyreg.pickle(ufunc, _ufunc_reduce)
-# Unclutter namespace (must keep _ufunc_reconstruct for unpickling)
+
+
+def _DType_reconstruct(scalar_type):
+    # This is a work-around to pickle type(np.dtype(np.float64)), etc.
+    # and it should eventually be replaced with a better solution, e.g. when
+    # DTypes become HeapTypes.
+    if scalar_type is None:
+        return dtype
+    return type(dtype(scalar_type))
+
```

Comment:
PEP 8 wants two blank line between top level function definitions.

Reply:
Fixed, had copied it from above, but rather changed it there (and moved the copyreg to be in one spot).